src/Components/IncidentFilter/IncidentFilter.test.js

```diff
@@ -1,14 +1,88 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { configure, shallow } from 'enzyme';
 import shallowToJSON from 'enzyme-to-json';
-
+import sinon from 'sinon';
+import Adapter from 'enzyme-adapter-react-16/build';
 import IncidentFilter from './IncidentFilter.Component';
 
+configure({ adapter: new Adapter() });
+
 describe('IncidentFilter component', () => {
+  let wrapper;
   it('should have all the IncidentFilter content', () => {
     const incidentFilter = shallow(<IncidentFilter />);
     const tree = shallowToJSON(incidentFilter);
     expect(tree.props.className).toEqual('filters-container');
     expect(tree.type).toEqual('div');
   });
+  it('should render all div elements', () => {
+    wrapper = shallow(<IncidentFilter />);
+    expect(wrapper.find('div').length).toEqual(4);
+  });
+  it('should render SelectField', () => {
+    wrapper = shallow(<IncidentFilter />);
+    expect(wrapper.find('SelectField').length).toEqual(2);
+  });
+  it('should render MenuItem', () => {
+    wrapper = shallow(<IncidentFilter />);
+    expect(wrapper.find('MenuItem').length).toEqual(8);
+  });
+  it('should render handleFlagChange', () => {
```

Comment:
I think the description can better. Maybe something like "should handle flag change" since this test case is checking if `handleFlagChange` was called. This applies to similar descriptions below.

Reply:
noted